src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs

```diff
@@ -152,6 +153,10 @@ public async Task<string> InvokeScriptAsync(CancellationToken ct, string script,
 			return await tcs.Task;
 		}
 
+		public IAsyncOperation<string> InvokeScriptAsync(string scriptName, IEnumerable<string> arguments) =>
+			InvokeScriptAsync(CancellationToken.None, scriptName, arguments?.ToArray()).AsAsyncOperation();
```

Comment:
It's best if you propagate the `CancellationToken` here.
```suggestion
			AsyncOperation.FromTask(async ct => InvokeScriptAsync(ct, scriptName, arguments?.ToArray()).AsAsyncOperation();
```